### ISSUE TITLE
[@typescript-eslint/no-magic-numbers] enum's are treated as "magic numbers"

### ISSUE THREAD

**Issue by radical-edo** [typescript-eslint/typescript-eslint#573]
**Repro**

```js
    'no-magic-numbers': 'off',
    '@typescript-eslint/no-magic-numbers': ['error', { ignoreNumericLiteralTypes: true }],

```
**Repro**
```ts
enum HttpStatus {
  Ok = 200
}
```

**Expected Result**
Don't treat it as a magic number. It will be used in the code somewhere as `HttpStatus.Ok` that's definitely a defined number.

**Actual Result**
The value `200` is treated as a magic number. Probably because to define a different starting point for `enum` is to use `=` which confuses linter.

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.9.0` |
| `@typescript-eslint/parser`        | `1.9.0` |
| `TypeScript`                       | `3.3.3333` |
| `ESLint`                           | `5.16.0` |
| `node`                             | `10.15.3` |
| `npm`                              | `6.4.1` |

**Comment by bradzacher** [comment]
See https://github.com/typescript-eslint/typescript-eslint/pull/543

**Comment by radical-edo** [comment]
Sweetness, can't wait to be merged.

**Comment by evgeniy-bond** [comment]
I am waiting for this merged too
